app/controllers/campaigns_controller.rb

```diff
@@ -12,8 +12,8 @@ class CampaignsController < ApplicationController
 
   def index
     campaigns = scope_list(Campaign)
-      .order(order_by)
-      .includes(:user, :creative, :organization)
+      .order(end_date: :desc)
+      .includes(:organization)
```

Comment:
Pretty sure the explicit `includes` for `user` and `creative` was from an effort to squash n+1 queries. Are you certain that we haven't reintroduced some with this change?

Reply:
I don't believe they are being used on the new campaign list